snuba/query/data_source/join.py

```diff
@@ -160,15 +160,6 @@ def get_alias_node_map(self) -> Mapping[str, IndividualNode[TSimpleDataSource]]:
             **self.right_node.get_alias_node_map(),
         }
 
-    def __post_init__(self) -> None:
```

Comment:
This was removed because of resolution: this check happens before resolving happens, so there are bunch of columns in the query that "don't exist".